WebRoot/messages/ZmMsg.properties

```diff
@@ -4567,7 +4569,10 @@ service.CODE_EXPIRED = The recovery email address verification code is expired.
 service.CONTACT_ADMIN = Something went wrong. Please contact your administrator.
 service.FEATURE_RESET_PASSWORD_SUSPENDED = Password reset feature is suspended.
 service.FEATURE_RESET_PASSWORD_DISABLED = Password reset feature is disabled.
+service.FEATURE_RESET_PASSWORD_MISMATCH = Sorry, your passwords do not match. Please try again.
 account.AUTH_FAILED = Verification code is invalid, please try again.
+account.INVALID_PASSWORD = Passwords must be at least 8 characters long.
```

Comment:
Does this show up if the password is missing a numeric character and/or symbol?

Reply:
Updated in commit 20098c4.